### ISSUE TITLE
Add "yes to all" when overwriting multiple files 

### ISSUE THREAD

**Issue by markknol** [fdorg/flashdevelop#1363]
When I copy/paste multiple files in the project manager, I get a dialog for each file I am overwriting. 
This is impractical when you just copied 50 files. It would be great if an option "yes to all". 
Also, I would expect that cancel would stop everything so acts like a "no to all".

![image](https://cloud.githubusercontent.com/assets/576184/19069121/ca7e7928-8a25-11e6-8e0d-03fd5940637c.png)

**Comment by Neverbirth** [comment]
Yeah, I agree on this one. I guess it's all due to laziness as the standard message box doesn't have the option, so the developer has to replicate all of the logic like icon, default titles, beep sound in some cases, accessibility etc, etc.

I guess there are good replacements out there.
